ksm/ksm_day3/k번째 수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+    let result = [];
+    
+    for (let command of commands) { // [253],[441],[173]
```

Comment:
```suggestion
    const result = [];
    
    for (const command of commands) { // [253],[441],[173]
```
@sim16065 
값이 변하지 않는다면 const를 사용해주세요.
항상 코드를 작성 할 때 1순위로 const를 사용하고, 값의 재할당이 발생할 때 let으로 변경하는 습관이 있어야 안정적인 코드를 작성하는데 도움이 됩니다

Reply:
좋은 방법인 것 같습니다.  다음 코테에 반영해보겠습니다!